BusinessCentral.LinterCop/LinterCopAnalyzers.resx

```diff
@@ -759,4 +759,13 @@
   <data name="Rule0071DoNotSetIsHandledToFalseDescription" xml:space="preserve">
     <value>The 'IsHandled' parameter must always be set to 'true' when handled. Avoid assigning 'false' or any value that may be evaluated as false.</value>
   </data>
+  <data name="Rule0072CheckProcedureDocumentationCommentTitle" xml:space="preserve">
+    <value>The documentation comment must match the procedure syntax.</value>
+  </data>
+  <data name="Rule0072CheckProcedureDocumentationCommentFormat" xml:space="preserve">
+    <value>The documentation comment does not match the procedure syntax.</value>
```

Comment:
It might be good to adjust the diagnostic messages so they better inform about the problem.
This message is very general. Better is to have the message describe that for example the documentation comment references a parameter that does not exist, and the other way around that a parameter is missing documentation.

Reply:
While I agree that a specific diagnostic message is always helpful, I also believe we need to weigh the complexity of the rule. Personally, I believe the rule is sufficient in its current form.

![image](https://github.com/user-attachments/assets/10bfcb3b-4438-4a62-ac0f-12772f3a9a01)